### PR TITLE
Upgrade fsd utils version

### PR DESCRIPTION
**Pull request by albertkol**
### Change description
Upgrade utils package.

### How to test
- packages seem to not have breaking changes
- tests are passing
- tested SSO on local with our test AD
- tested partly magic link (but I struggled to do it all)

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -50,4 +50,5 @@ prance
 #-----------------------------------
 #   Other
 #-----------------------------------
-beautifulsoup4==4.12.2
+beautifulsoup4
+greenlet
```
Comment (samuelhwilliams) [inline requirements.in:54]: can you include the comment here please

---

arguably we then _only_ need it here and could remove it from data-store I think

Comment (albertkol) [reply requirements.in]: We need it, no? The reason we list it is because `pip-compile` is acting differently on different OS's.

Comment (samuelhwilliams) [reply requirements.in]: Yes. Can you include the comment explaining that here? We added one in data store

Comment (albertkol) [reply requirements.in]: Added

**Review by samuelhwilliams — APPROVED** [review]

**Review by hamzabinkhalid — APPROVED** [review]

**Pull request merged**